docs/concepts/what-is-a-workflow-definition.md

```diff
@@ -62,7 +62,7 @@ Before the Timer is up, we change the Workflow Definition to the following seque
 When the Timer fires, the next Workflow Task will cause the Workflow Function to re-execute.
 The first Command the Worker sees would be be ScheduleActivityTask Command, which wouldn't match up to the expected [TimerStarted](/references/events/#timerstarted) Event.
 
-The Workflow Execution would fail, and return the non-determinism error.
+The Workflow Execution would fail, and return a non-deterministic error.
```

Comment:
That edit changes the meaning, I assert. @flossypurse, what's a better way to cite an error thrown because of non-determinism?

Reply:
Across languages there are several things you could see that could be categorized as a "non-deterministic error".

So for now - this satisfies I think.